src/simulation/dynamics/reactionWheels/reactionWheelStateEffector.cpp

```diff
@@ -310,8 +310,30 @@ void ReactionWheelStateEffector::computeDerivatives(double integTime, Eigen::Vec
         }
 		if (RWIt->RWModel == BalancedWheels || RWIt->RWModel == JitterSimple) {
 			OmegasDot(RWi,0) = (RWIt->u_current + RWIt->frictionTorque)/RWIt->Js - RWIt->gsHat_B.transpose()*omegaDotBNLoc_B;
+
+            // Check for numerical instability due to excessive wheel acceleration
+            const double MAX_WHEEL_ACCELERATION = 1.0e6; // rad/s^2, adjust as needed
```

Comment:
I would make this a private class variable that defaults to 1.0e6.  Then we need the usual setter and getter functions so a user could pull this info or set it.  Adjust module documentation to describe this variable and how to read/set it. 
This way we don't expect the user to read code to find this behavior, what its default value is, how to set it.